### PR TITLE
improve Clustering impl

### PR DESCRIPTION
**Pull request by ava57r**

### CODE REVIEW

**Comment by ava57r** [comment]
faiss PR https://github.com/facebookresearch/faiss/pull/1945

**Comment by ava57r** [comment]
#33

File: src/cluster.rs [hunk 1]
```diff
@@ -305,6 +321,11 @@ impl Clustering {
         unsafe { faiss_Clustering_spherical(self.inner) != 0 }
     }
 
+    /** Getter for the `int_centroids` property of `Clustering`. */
+    pub fn int_centroids(&self) -> bool {
+        unsafe { faiss_Clustering_int_centroids(self.inner) != 0 }
+    }
+
     /** Getter for the `update_index` property of `Clustering`. */
```
Comment (Enet4) [inline src/cluster.rs:329]:
```suggestion
    /// Getter for the `update_index` property of `Clustering`.
```

**Review by Enet4 — COMMENTED** [review]
Thank you @ava57r, it seems mostly OK, but could you also include a short piece of documentation per new method? It is something previously overlooked, and now the various methods are starting to become overwhelming, so documenting them could help users find what they need through search.

More suggestions inline.

**Review by Enet4 — APPROVED** [review]
Great! Thanks!

**Pull request merged**